.jshintrc

```diff
@@ -0,0 +1,26 @@
+{
+	"boss": true,
+	"curly": true,
+	"eqeqeq": true,
+	"eqnull": true,
+	"es3": true,
+	"expr": true,
+	"immed": true,
+	"noarg": true,
+	"nonbsp": true,
+	"onevar": true,
+	"quotmark": "single",
+	"trailing": true,
+	"undef": true,
+	"unused": true,
+
+	"browser": true,
+
+	"globals": {
+		"_": false,
+		"Backbone": false,
+		"jQuery": false,
+		"JSON": false,
+		"wp": false
+	}
+}
```

Comment:
We can pull this file in with `wget https://develop.svn.wordpress.org/trunk/.jshintrc`. No need to create disparity between _s and how Core does things. See [this](https://github.com/WordPress/twentysixteen/blob/master/.travis.yml#L75).